### PR TITLE
Upgrade polyfill-library to 3.111.0

### PR DESCRIPTION
**Pull request by jdeniau**
🤞

### CODE REVIEW

File: server/routes/v3/polyfill.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ const polyfillio_3_103_0 = require("polyfill-library-3.103.0");
 const polyfillio_3_104_0 = require("polyfill-library-3.104.0");
 const polyfillio_3_108_0 = require("polyfill-library-3.108.0");
 const polyfillio_3_109_0 = require("polyfill-library-3.109.0");
+const polyfillio_3_110_1 = require("polyfill-library-3.110.1");
```
Comment (JakeChampion) [inline server/routes/v3/polyfill.js:34]: This new variable will need to be added into the array on line 104 of this file 👍

**Review by JakeChampion — CHANGES_REQUESTED** [review]
Thanks for much for this @jdeniau, there is one blocking change which will need to be done before this can be merged and released 👍 

https://github.com/Financial-Times/polyfill-service/pull/2674/files#r835157065

File: server/routes/v3/polyfill.js (hunk 1 above)
Comment (jdeniau) [reply server/routes/v3/polyfill.js:34]: Done. 
It seems that sometime the older version is not set in the files so I tried to add 3.110.1 everywhere, but I don't know if it is really expected ¯\_(ツ)_/¯

**Review by JakeChampion — APPROVED** [review]

File: server/routes/v3/polyfill.js (hunk 1 above)
Comment (JakeChampion) [reply server/routes/v3/polyfill.js:34]: That would have been an error from me 😔 thank you for finding and fixing all those

**Comment by jdeniau** [comment]
@JakeChampion Is there something that I can do to release this onto the cdn ? 🤔 (It seems that is still the 3.110.1 in production)

**Comment by JakeChampion** [comment]
> @JakeChampion Is there something that I can do to release this onto the cdn ? 🤔 (It seems that is still the 3.110.1 in production)

I've started the process to release this to production now 👍

**Pull request merged**